packages/amazonq/.changes/next-release/Bug Fix-4b44becd-f933-48b3-9e77-62da8504969a.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "refactor error message and include backfill for error name"
```

Comment:
Thanks for including a changelog. Please review the [changelog guidelines](https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog).